frontend/tasks/content.js

```diff
@@ -32,6 +39,34 @@ gulp.task('content-extract-experiment-strings', () =>
     .pipe(buildExperimentsFTL())
     .pipe(gulp.dest('./locales/en-US')));
 
+function populateRSSFeed(newsUpdates) {
+  return new Feed({
+    title: 'Test Pilot News Updates',
+    description: 'News Updates for Test Pilot experiments',
+    id: 'https://blog.mozilla.org/testpilot',
+    link: 'https://blog.mozilla.org/testpilot',
+    favicon: 'https://testpilot.firefox.com/static/images/favicon.ico',
+    feedLinks: {
+      rss: 'https://testpilot.firefox.com/feed.rss',
+      atom: 'https://testpilot.firefox.com/feed.atom',
+      json: 'https://testpilot.firefox.com/feed.json'
+    },
+    author: {
+      name: 'Mozilla',
+      link: 'https://testpilot.firefox.com'
+    },
+    feed: newsUpdates.map(post => {
+      return {
+        title: post.title,
+        id: post.link,
+        link: post.link,
+        date: post.published,
+        content: marked(post.content)
```

Comment:
Despite my earlier example content mentioning Markdown - the news update component doesn't actually render the content using Markdown. [It's just text.](https://github.com/mozilla/testpilot/blob/master/frontend/src/app/components/UpdateList.js#L39) If we do this in the feeds here, we should add it to the `Update` component [using the `ReactMarkdown` component we use in the `pages` tasks](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/pages.js#L256)